src/bp-forums/activity.php

```diff
@@ -169,12 +169,8 @@ private function setup_filters() {
 			// Filter for single topic.
 			add_filter( 'bbp_is_single_topic', array( $this, 'set_single_topic' ) );
 
-			// Filter discussion.
-			add_filter( 'bp_get_activity_content_body', array( $this, 'before_activity_content' ), 10, 2 );
```

Comment:
Please don't remove this filter.

Reply:
Back to the previous one.
https://github.com/buddyboss/buddyboss-platform/pull/2635/commits/7593d31b8fde819594390a94b715746fd0abbee1